worlds/ff6wc/Rom.py

```diff
@@ -542,8 +542,8 @@
     "Phantom Train Third Car Far Right Chest": (0x1E47, 2, 111),
     "Phoenix Cave Lower Cavern East Pool Island": (0x1E56, 0, 192),
     "Phoenix Cave Lower Cavern East Pool Bridge": (0x1E55, 6, 193),
-    "Phoenix Cave Lower Cavern Spikes": (0x1E55, 7, 194),
-    "Phoenix Cave Lower Cavern Rock Jumping": (0x1E57, 6, 195),
+    "Phoenix Cave Lower Cavern Spikes": (0x1E57, 6, 195), # Lower Cavern Spikes and Lower Cavern Rock Jumping were swapped
+    "Phoenix Cave Lower Cavern Rock Jumping": (0x1E55, 7, 194), # Lower Cavern Rock Jumping and Lower Cavern Spikes were swapped
     "Phoenix Cave Lower Cavern Cool Lava": (0x1E55, 3, 191),
```

Comment:
I'm trying to make sure I understand what these comments are saying.
Have these 2 locations been wrong in our 1.2 apworld for a long time? So this change isn't related to the 1.4.2 change?

Reply:
That's correct, the descriptions of the 2 chests were wrong & swapped. I noticed this while playing and decided to fix it.